.circleci/config.yml

```diff
@@ -202,7 +225,18 @@ workflows:
   version: 2
   build:
     jobs:
-      - lint_&_docs
+      - lint_&_docs:
+          filters:
+            branches:
+              only: /.*/
+      - docs-deploy:
+          requires:
+            - lint_&_docs
+          filters:
+            tags:
+              only: /^v\d+\.\d+\.\d+$/
```

Comment:
Is it gonna deploy when new release tag is published?

Reply:
what is the recommended way to deploy it manually in case of failure?

Reply:
I changed it so it only deploys on x.x.0 tags, so it will run on semver minor changes like 0.3.0 and 0.4.0 but not 0.4.1. In order to deploy it manually we will have to deploy to the gh-pages branch manually. I can create a lerna task for that.

Reply:
Ok got it. For now, we have to deploy it manually right? otherwise we need to wait till `0.4.0` release?

Reply:
yes